### ISSUE TITLE
Connect AddressAndPort

### ISSUE THREAD

**Issue by xxAtrain223** [getnamo/socketio-client-ue4#7]
TL;DR [Line 30](https://github.com/getnamo/socketio-client-ue4/blob/master/Source/SocketIOClient/Private/SocketIOClientComponent.cpp#L30) should be changed to `if (InAddressAndPort.IsEmpty())`.

I've been playing around with the plugin and I found a pretty major (1 line) bug.
The default usage for the Connect function is to use `http://localhost:3000`.
To do this, you check if the `InAddressAndPort` is not empty. Problem is, on [Line 30](https://github.com/getnamo/socketio-client-ue4/blob/master/Source/SocketIOClient/Private/SocketIOClientComponent.cpp#L30), you use the default `AddressAndPort` which is hardcoded to use the above address, which is never empty. 
Therefore, the plugin only connects to the default address.

**Comment by getnamo** [comment]
Good catch, merged.
